plugins/modules/net_tools/nmcli.py

```diff
@@ -77,6 +77,12 @@
             - Use the format C(192.0.2.1).
             - This parameter is mutually_exclusive with never_default4 parameter.
         type: str
+    gw4_ignore_auto:
+        description:
+            - Ignore automatically configured IPv4 routes.
+        type: bool
+        default: no
```

Comment:
```suggestion
        default: false
```
Please use `true`/`false` for YAML booleans.